pandas/core/missing.py

```diff
@@ -404,7 +429,13 @@ def _from_derivatives(xi, yi, x, order=None, der=0, extrapolate=False):
     return m(x)
 
 
-def _akima_interpolate(xi, yi, x, der=0, axis=0):
+def _akima_interpolate(
+    xi: ArrayLike,
+    yi: ArrayLike,
```

Comment:
the docstring says ArrayLike, but im guessing this wouldnt work on EAs

Reply:
I guess not since this is a  wrapper for `scipy.interpolate.Akima1DInterpolator` (and mypy works with both of these annotated as`np.ndarray`)